packages/sdk-js/src/GrowthBook.ts

```diff
@@ -726,20 +734,33 @@ export class GrowthBook<
               id: experiment.key,
             }
           );
+          this._unsetAntiFlicker();
           return result;
         }
         this._redirectedUrl = url;
         const navigate = this._getNavigateFunction();
         if (navigate) {
           if (isBrowser) {
-            this._setAntiFlicker();
-            window.setTimeout(() => {
+            this._setAntiFlicker(true);
+            // Wait for the possibly-async tracking callback, bound by min and max delays
+            Promise.all([
+              promiseTimeout(
+                this._activeTrackingCall,
+                this._ctx.maxNavigateDelay ?? 10000
```

Comment:
10 seconds seems way too long here.  I was thinking it would be more like 1 second

Reply:
sure. i'm looking at it thru the lens of the folks who have been experiencing major SRM issues due to a large number of tracking calls taking over 1-2 secs